build_scripts/windows/scripts/build.cmd

```diff
@@ -180,6 +179,8 @@ if exist "%PROPAGATE_ENV_CHANGE_DIR%\propagate_env_change.exe" (
 echo Building MSI...
 msbuild /t:rebuild /p:Configuration=Release %REPO_ROOT%\build_scripts\windows\azure-cli.wixproj
 
+if %errorlevel% neq 0 goto ERROR
```

Comment:
Explicitly check if the previous command succeeds, otherwise even if `msbuild` fails, CI shows ✅.